conf/nfcore_license.config

```diff
@@ -0,0 +1,13 @@
+//Profile config names for nf-core/configs
+params {
+    config_profile_description = 'Sentieon license set up for nf-core by nf-core/configs'
+    config_profile_contact = 'Maxime U Garcia (@maxulysse)'
+    config_profile_url = 'https://www.sentieon.com'
+}
+
+process {
+    withLabel: 'sentieon' {
+        ext.sentieon_auth_mech_base64 = secrets.SENTIEON_AUTH_MECH_BASE64 ?: ''
```

Comment:
```suggestion
        ext.sentieon_auth_mech_base64 = secrets.SENTIEON_AUTH_MECH_BASE64
```

Reply:
let's merge without and we can improve on later, that's the whole point behind this implementation